odoo/addons/base/migrations/12.0.1.3/openupgrade_analysis.txt

```diff
@@ -74,7 +74,9 @@ DEL ir.ui.view: base.res_request_link_search_view
 DEL ir.ui.view: base.res_request_link_tree-view
 NEW res.country: base.xk (noupdate)
 NEW res.lang: base.lang_en_CA
-NEW res.partner: base.partner_admin (noupdate)
 DEL res.request.link: base.req_link_partner (noupdate)
 NEW res.users: base.template_portal_user_id
+# Done: moved XMLID of this user from auth_signup
 NEW res.users: base.user_admin (noupdate)
+NEW res.partner: base.partner_admin (noupdate)
+# Done: split up this user and partner from what is now the 'system' user
```

Comment:
I think this kind of comments should go into the `openupgrade_analysis_work.txt` file instead.

Reply:
Indeed, done!

Reply:
But also I think you should revert the changes done in this `openupgrade_analysis.txt`